examples/ui.apps/src/main/content/jcr_root/apps/cif-components-examples/clientlibs/venia-theme/.content.xml

```diff
@@ -0,0 +1,5 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<jcr:root xmlns:cq="http://www.day.com/jcr/cq/1.0" xmlns:jcr="http://www.jcp.org/jcr/1.0"
+    jcr:primaryType="cq:ClientLibraryFolder"
+    allowProxy="{Boolean}true"
+    categories="[venia-theme]"/>
```

Comment:
Shouldn't we use a different name instead of "Venia"? 
Before you ask, I don't have another suggestion :)

Reply:
I'd keep `venia`, we do really use the **venia** theme from the archetype. ;-)